causal_testing/json_front/json_class.py

```diff
@@ -188,7 +225,9 @@ def _execute_test_case(self, causal_test_case: CausalTestCase, test: Iterable[Ma
             logger.warning("   FAILED- expected %s, got %s", causal_test_case.expected_causal_effect, result_string)
         return failed
 
-    def _setup_test(self, causal_test_case: CausalTestCase, test: Mapping) -> tuple[CausalTestEngine, Estimator]:
+    def _setup_test(
+        self, causal_test_case: CausalTestCase, test: Mapping, conditions: list[str] = None
```

Comment:
Might be nice to add the new `conditions` param to the docstring, but it is optional as this is a private method that users won't be interacting with. 

I believe that pylint does not pick this up as it is private

Reply:
Good spot. Fixed